tailwind.config.js

```diff
@@ -261,7 +262,7 @@ module.exports = {
             '6xl': '72rem',
             full: '100%',
             screen: '100vh',
-            '3/10': '30%',
+            '3/10': '30vw',
```

Comment:
don't change an existing class, you do not know who else is using that.
Also, since **screen** is already operating in vw unit and the rest (s,m,l,xl..) are in rem, maybe make a new as **3/10-screen**

Reply:
Actually, **3/10** was added by my, and wasn't present already. This is me editing my own custom tag. 

I will look into your suggestion. I should have actually commented why I changed from % to vw, because as of now, I cannot recall.